elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -109,16 +102,16 @@ $pfe-cta__arrow-size: 0.313em;
 }
 
 ::slotted(a) {
-  display: block;
-  color: #{pfe-local(Color)};
+  display: inline;
+  color: #{pfe-local(Color, #003366)} !important;
 
   padding: #{pfe-local(Padding)};
   text-decoration: none;
 }
 
 ::slotted(a:focus) {
   outline: none !important;
-  color: #{pfe-local(Color--focus)};
+  color: #{pfe-local(Color--focus)} !important;
 }
 
 :host(:focus-within) {
```

Comment:
@castastrophe @chrisdo1  the polyfill for IE11 trips over the `focus-within` selector. If its present, the whole component does not upgrade. Kyle suggested we remove it and instead put the arrow in the light dom, POC 
```
class PfeCTa extends HTMLElement {
	constructor() {
  	this._hasArrow = false;	
  }
  
	connectedCallback() {
  	super.connectedCallback()
    
    const arrow = `<svg whatever>`;
    const fragment = document.createDocumentFragment();
    fragment.innerHTML = arrow;
    
    if (!this._hasArrow) {
    	this.appendChild(fragment);
      this._hasArrow = true
    }
  }
}
```